### PR TITLE
individual review page, css seperated

### PR DESCRIPTION
**Pull request by Oathian**

### CODE REVIEW

File: src/components/GamesList.jsx [hunk 1]
```diff
@@ -19,7 +20,7 @@ const GamesList = () => {
             {reviews.map((review) => {
                 return(
                     <section className="GamesList__GamesCard" key={review.review_id}>
-                        <h3 className="GamesCard__h3">{review.title}</h3>
+                        <Link className="GamesCard__h3" key={review.review_id} to={`/reviews/${review.review_id}`}><h3>{review.title}</h3></Link>
```
Comment (DCrawley94) [inline src/components/GamesList.jsx:23]: It's up to you but you could wrap the link around the whole section. That way users can click the card itself to naviagte :)

File: src/components/ReviewsByCategory.jsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ const ReviewsByCategory = () => {
             {reviews.map((review) => {
                 return(
                     <section className="GamesList__GamesCard" key={review.review_id}>
-                        <h3 className="GamesCard__h3">{review.title}</h3>
+                        <Link className="GamesCard__h3" key={review.review_id} to={`/reviews/${review.review_id}`}><h3>{review.title}</h3></Link>
```
Comment (DCrawley94) [inline src/components/ReviewsByCategory.jsx:26]: The reviews by category component is almost identical to the games list component. Can you figure out a way to combine the two. You've done 90% of the work already :)

**Review by DCrawley94 — CHANGES_REQUESTED** [review]
This is good but there's a way you can DRY up your code by not having identical components. You could also dry up your API functions by using the axios params object. Have a think about it and feel free to NC Help if you're not sure :)

File: src/components/GamesList.jsx (hunk 1 above)
Comment (Oathian) [reply src/components/GamesList.jsx:25]: I was considering this but I think it might break something I'm considering adding later

**Pull request merged**